wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -915,6 +916,11 @@ class KotlinGenerator private constructor(
             if (constant.documentation().isNotBlank()) {
               addKdoc("%L\n", constant.documentation())
             }
+            if ("true" == constant.options().get(ENUM_DEPRECATED)) {
+              addAnnotation(AnnotationSpec.builder(Deprecated::class)
+                  .addMember("message = %S", "${constant.name()} is deprecated")
```

Comment:
Not sure it adds relevant information to what the annotation already provides.

Reply:
Yeah, but `message` is mandatory in `Deprecated`, so you have to pass something. This is consistent with what we pass in for deprecated fields.